schoolopy/main.py

```diff
@@ -2003,14 +2085,29 @@ def get_message(self, message_id):
 
     # Implement search, resource collections, resource templates
 
+    def _like_item(self, pathItems, unlike=False):
+        """
+        Post a like request
+
+        :param pathItems: The path with values to POST to
+        :param unlike: Whether to unlike or like (Default like)
+        :return: Number of likes on the object.
+        """
+        try:
+            return self._post(pathItems, json.dumps({'like_action': True and (not unlike)}))['likes']
+        except TypeError:
+            return "The status you posted is already present."
```

Comment:
Not great practice to return a string describing the error. Not to mention that I don't think this describes the error that would happen.

Instead, you should just throw the error. By returning a string to a function which probably isn't expecting one, you might break a user's script.